### PR TITLE
zh: update doc for tiflash on eks and ack

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)

update doc for tiflash on eks and ack
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ category: how-to
     operator_version = "v1.1.0-rc.1"
     ```
 
+    如果需要在集群中部署 TiFlash，需要在 `terraform.tfvars` 中设置 `create_tiflash_node_pool = true`，也可以设置 `tiflash_count` 和 `tiflash_instance_type` 来配置 TiFlash 节点池的节点数量和实例类型，`tiflash_count` 默认为 `2`，`tiflash_instance_type` 默认为 "ecs.i2.2xlarge"。
```
Comment (ran-huang) [inline zh/deploy-on-alibaba-cloud.md:93]:
```suggestion
    如果需要在集群中部署 TiFlash，需要在 `terraform.tfvars` 中设置 `create_tiflash_node_pool = true`，也可以设置 `tiflash_count` 和 `tiflash_instance_type` 来配置 TiFlash 节点池的节点数量和实例类型，`tiflash_count` 默认为 `2`，`tiflash_instance_type` 默认为 `ecs.i2.2xlarge`。
```

File: zh/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -101,6 +101,8 @@ eks_name = "my-cluster"
 operator_version = "v1.1.0-rc.1"
 ```
 
+如果需要在集群中部署 TiFlash，需要在 `terraform.tfvars` 中设置 `create_tiflash_node_pool = true`，也可以设置 `cluster_tiflash_count` 和 `cluster_tiflash_instance_type` 来配置 TiFlash 节点池的节点数量和实例类型，`cluster_tiflash_count` 默认为 `2`，`cluster_tiflash_instance_type` 默认为 "i3.4xlarge"。
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:104]:
```suggestion
如果需要在集群中部署 TiFlash，需要在 `terraform.tfvars` 中设置 `create_tiflash_node_pool = true`，也可以设置 `cluster_tiflash_count` 和 `cluster_tiflash_instance_type` 来配置 TiFlash 节点池的节点数量和实例类型，`cluster_tiflash_count` 默认为 `2`，`cluster_tiflash_instance_type` 默认为 `i3.4xlarge`。
```

File: zh/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -159,10 +161,35 @@ region = us-west-21
 
     参考 [API 文档](api-references.md)和[集群配置文档](configure-cluster-using-tidbcluster.md)完成 CR 文件配置。
 
+    如果要部署 TiFlash，可以在 db.yaml 中配置 `spec.tiflash`，例如：
+
+    ```yaml
+    spec
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:167]:
```suggestion
    spec:
```

File: zh/scale-a-tidb-cluster.md [hunk 4]
```diff
@@ -14,8 +14,55 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
...
+
+    ```shell
+    curl 127.0.0.1:2379/pd/api/v1/config/rules/group/tiflash | grep count
+    ```
+
+    输出结果中 `count` 的最大值就是所有数据表的最大副本数 N。
+  
+3. 回到 `port-forward` 命令所在窗口，按 <kbd>Ctrl</kbd>+<kbd>C</kbd> 停止 `port-forward`。
+
+4. 如果缩容 TiFlash 后，TiFlash 集群剩余 Pod 数大于等于所有数据表的最大副本数 N，直接进行下面第 6 步。如果缩容 TiFlash 后，TiFlash 集群剩余 Pod 数小于所有数据表的最大副本数 N，参考[访问 TiDB 集群](access-tidb.md)，连接到 TiDB 服务，并针对所有副本数大于集群剩余 TiFlash Pod 数的表执行如下命令：
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md:47]:
```suggestion
4. 如果缩容 TiFlash 后，TiFlash 集群剩余 Pod 数大于等于所有数据表的最大副本数 N，直接进行下面第 6 步。如果缩容 TiFlash 后，TiFlash 集群剩余 Pod 数小于所有数据表的最大副本数 N，参考[访问 TiDB 集群](access-tidb.md)的步骤连接到 TiDB 服务，并针对所有副本数大于集群剩余 TiFlash Pod 数的表执行如下命令：
```

**Comment by DanielZhangQD** [comment]
@ran-huang comments addressed, PTAL.

**Pull request merged**